ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -212,74 +186,15 @@ export const Homepage = fp.flow(withUserProfile(), withNavigation, withRouter)(c
     clearTimeout(this.timer);
   }
 
-  // This method is part of the eRA Commons linkage flow. The "return-url" sent to the Shibboleth
-  // server is set to "nih-callback", which is configured in our router to point to the homepage
-  // component. If we reach this component with a 'token' query parameter, we will attempt to
-  // store that token in Terra.
-  async validateNihToken() {
-    const token = (new URL(window.location.href)).searchParams.get('token');
-    if (token) {
-      this.setState({eraCommonsLoading: true});
-      try {
-        const profileResponse = await profileApi().updateNihToken({ jwt: token });
-        if (profileResponse.eraCommonsLinkedNihUsername !== undefined) {
-          this.setState({eraCommonsLinked: true});
-        }
-      } catch (e) {
-        this.setState({eraCommonsError: 'Error saving NIH Authentication status.'});
-      }
-    }
-  }
-
-  async validateRasLoginGovLink() {
-    const authCode = (new URL(window.location.href)).searchParams.get('code');
-    const redirectUrl = buildRasRedirectUrl();
-    if (authCode) {
-      this.setState({rasLoginGovLoading: true});
-      try {
-        const profileResponse = await profileApi().linkRasAccount({ authCode, redirectUrl });
-        if (profileResponse.rasLinkLoginGovUsername !== undefined) {
-          this.setState({rasLoginGovLinked: true});
-        }
-        if (profileResponse.eraCommonsLinkedNihUsername !== undefined) {
-          this.setState({eraCommonsLinked: true});
-        }
-      } catch (e) {
-        this.setState({rasLoginGovLinkError: 'Error saving RAS Login.Gov linkage status.'});
-        this.setState({rasLoginGovLoading: false});
-      }
-    }
-    // Cleanup parameter from URL after linking.
-    window.history.replaceState({}, '', '/');
-  }
-
   setFirstVisit() {
     this.setState({firstVisit: true});
     profileApi().updatePageVisits({ page: this.pageId});
   }
 
-  getRegistrationTasksMap() {
-    return getRegistrationTasksMap(this.props.navigate);
-  }
-
+  // TODO reload profile?
```

Comment:
Can you please create a ticket for each TODO and put the JIRA ticket number in the comment?